### ISSUE TITLE
High Priority: text direction

### ISSUE THREAD

**Issue by mMoliere** [majlis-erc/manuForma#476]
@wsalesky A number of elements come with xml:lang. We want this attribute to control the direction of text (RTL or LTR) in manuForma.

The following values shall instruct manuForma to display text RTL:

- ar
- he
- he-Latn
- ar-Latn
- ar-Hebr

And this value set the display to LTR:

- en

**Comment by wsalesky** [comment]
@mMoliere Do you mean the display of the input boxes?

**Comment by mMoliere** [comment]
@wsalesky Yes, it is about the way text is being displayed in the input fields.
